### PR TITLE
reorganize package to support multiple clients

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: v0/examples/example.go [hunk 1]
```diff
@@ -36,7 +36,8 @@ func main() {
 			createTuple(tuple(aDoc("owner"), anOwner)),
 			createTuple(tuple(aDoc("contributor"), anEditor)),
 			createTuple(tuple(aDoc("viewer"), aViewer)),
-		}})
+		},
```
Comment (josephschorr) [inline v0/examples/example.go:39]: Why this change?

**Review by josephschorr — APPROVED** [review]
LGTM

File: v0/examples/example.go (hunk 1 above)
Comment (jzelinskie) [reply v0/examples/example.go:39]: gofumpt
